### DISCUSSION TITLE
Getter method is changing property order after return

### DISCUSSION THREAD

**Discussion by dutchman1990** [sequelize/sequelize#14938] [Help & Questions]
I have added getter for createdAt column to get datetime in dd-mm-yyyy format, but after changing that specific field is on top of each object. 

> Befor addition of getter method

`[
    {
        "state_id": 2,
        "state_name": "West Bengal",
        "short_name": "WB",
        "created_at": "17-10-2021 10:30:33",
    }
]`

> After addition of getter method

`[
    {
        "created_at": "17-10-2021 10:30:33",
        "state_id": 2,
        "state_name": "West Bengal",
        "short_name": "WB"
    }
]`

is there any way to set to the original order?

**Comment by fzn0x** [comment]
Did the changed order causing an issue for your application?

**Reply by dutchman1990** [reply]
Nope, but it should not happen IMO. Our postman samples were saved in order before implementing getter method and now app developers aren't seeing them until they are looking closely at the response.
